content/articles/restoring-domain.md

```diff
@@ -18,8 +18,9 @@ The registrar applies a mandatory restore fee. The final price of the restore wi
 
 ##### To restore a domain
 
-1. Log in to DNSimple with your user credentials.
-1.  On the header, click the <label>Domains</label> tab, locate the relevant domain, and click on the name to access the domain page.
+1.  Log in to DNSimple with your user credentials.
```

Comment:
This seems like the kind of thing we could remove from _all_ of our instructions. 😅

```suggestion
```